qa/suites/rbd/nvmeof/basic/workloads/nvmeof_initiator.yaml

```diff
@@ -1,13 +1,14 @@
 tasks:
 - nvmeof:
     client: client.0
-    version: default  # "default" is the image cephadm defaults to; change to test specific nvmeof images, example "latest"
+    gw_image: quay.io/ceph/nvmeof:1.2 # "default" is the image cephadm defaults to; change to test specific nvmeof images, example "latest"
```

Comment:
@caroav this needs to be changed back to "default" before we merge this PR with HA PR (reason: teuthology should be testing the existing code instead of hard-coding the tag in tests). 

For "default" to work, this line of code needs to be changed in HA PR to "1.2" (or "1.2.9"):
https://github.com/ceph/ceph/blob/a7d580f9e755ecd6ad49d1d84fb2739a4bf0e70c/src/pybind/mgr/cephadm/module.py#L123

Reply:
@baum, what do you think about `DEFAULT_NVMEOF_IMAGE` for the HA branch? Your insights would be helpful.